### PR TITLE
start decompiling field_control_avatar.c

### PR DESCRIPTION
**Pull request by camthesaxman**

### CODE REVIEW

File: src/field_control_player_avatar.c [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#include "global.h"
```
Comment (YamaArashi) [inline src/field_control_player_avatar.c]: what is this?

Comment (camthesaxman) [inline src/field_control_player_avatar.c]: I'm including global.h like almost all of our other C files do.

Comment (YamaArashi) [inline src/field_control_player_avatar.c]: what is this file?

Comment (camthesaxman) [inline src/field_control_player_avatar.c]: Oh, that file. I accidentally named it the wrong thing and forgot to delete it.

**Pull request merged**